valhalla/mjolnir/osmaccessrestriction.h

```diff
@@ -59,7 +59,7 @@ class OSMAccessRestriction {
   uint64_t value_;
 
   struct Attributes {
-    uint16_t type_ : 4;
+    uint16_t type_ : 6;
```

Comment:
lets put a comment here that this explains whats going on:

```suggestion
    uint16_t type_ : 6;  // interpreted as AccessType but stored in AccessRestriction::type_
```